mcm/json_layer/mccm.py

```diff
@@ -65,3 +65,26 @@ def get_editable(self):
             for key in self._json_base__schema:
                 editable[key] = False
         return editable
+
+    @staticmethod
+    def get_mccm_by_generated_chain(chain_id):
+        mccms_db = database('mccms')
+        __query = mccms_db.construct_lucene_query({'generated_chains' : chain_id})
+        result = mccms_db.full_text_search('search', __query, page=-1)
+        try:
+            return mccm(json_input=result[0])
+        except Exception as ex:
+            return None
```

Comment:
Since it is a static method, it doesn't have self.logger, I put the logger messages when i call this method